frontend/src/views/ensemble/SingleNote.tsx

```diff
@@ -27,6 +29,9 @@ let instrumentSampler = fluteSampler;
 
 const SingleNote: FC<SingleNoteProps> = ({ beatNumber, pitch }) => {
   const palette = useRecoilValue(paletteAtom);
+  const [currentEnsemble, setCurrentEnsemble] = useRecoilState(ensembleAtom);
+  // const userID = useRecoilValueLoadable(userIDSelector).contents;
```

Comment:
Let's delete this comment, unless you have a reason you'd like to keep this comment around?